### ISSUE TITLE
RayOnSpark raises zombie exception

### ISSUE THREAD

**Issue by songhappy** [intel-analytics/analytics-zoo#3042]
environment: MacOS 10.15.7, python3.6
code
```
from zoo.orca import init_orca_context, stop_orca_context
import ray
from zoo.ray import RayContext
sc = init_orca_context(cluster_mode="local", cores=8, memory="20g")
ray_ctx = RayContext(sc=sc, object_store_memory="5g")
ray_ctx.init()
ray_ctx.stop()
sc.stop()
```
error message
  ray_ctx.init()
  File "/Users/guoqiong/opt/anaconda3/envs/py36tf2/lib/python3.6/site-packages/zoo/ray/raycontext.py", line 473, in init
    kill_redundant_log_monitors(self._address_info["redis_address"])
  File "/Users/guoqiong/opt/anaconda3/envs/py36tf2/lib/python3.6/site-packages/zoo/ray/raycontext.py", line 70, in kill_redundant_log_monitors
    cmdline = subprocess.list2cmdline(proc.cmdline())
  File "/Users/guoqiong/opt/anaconda3/envs/py36tf2/lib/python3.6/site-packages/psutil/__init__.py", line 675, in cmdline
    return self._proc.cmdline()
  File "/Users/guoqiong/opt/anaconda3/envs/py36tf2/lib/python3.6/site-packages/psutil/_psosx.py", line 342, in wrapper
    return fun(self, *args, **kwargs)
  File "/Users/guoqiong/opt/anaconda3/envs/py36tf2/lib/python3.6/site-packages/psutil/_psosx.py", line 429, in cmdline
    return cext.proc_cmdline(self.pid)
  File "/Users/guoqiong/opt/anaconda3/envs/py36tf2/lib/python3.6/contextlib.py", line 99, in __exit__
    self.gen.throw(type, value, traceback)
  File "/Users/guoqiong/opt/anaconda3/envs/py36tf2/lib/python3.6/site-packages/psutil/_psosx.py", line 374, in catch_zombie
    raise ZombieProcess(proc.pid, proc._name, proc._ppid)
psutil.ZombieProcess: psutil.ZombieProcess process still exists but it's a zombie (pid=1274, name='uname')

**Comment by qiyuangong** [comment]
It's a psutil issue on macOS. This error is quite similar to #2832 (fixed by #2856).

Have you tried latest Zoo nighty-build on macOS?

**Comment by songhappy** [comment]
this error is from pip install, good to know that it is fixed, close this one now.
